src/server/actions/Event.ts

```diff
@@ -55,3 +57,17 @@ export async function createEvent(
     throw new CMError(CMErrorType.InternalError);
   }
 }
+
+export async function deleteEvent(eventId: string): Promise<void> {
+  try {
+    await dbConnect();
+
+    EventVolunteerSchema.deleteMany({
```

Comment:
You need to use the await keyword here. Otherwise I believe everything looks good.

Reply:
I you are interested, this website explains why and how async/await works with mongo. https://mongoosejs.com/docs/async-await.html
If you have any question let me know.